### PR TITLE
Explain how to add `from_transport` on `messenger.message_handler` tag

### PR DESCRIPTION
**Pull request by ruudk**
See https://github.com/symfony/symfony/pull/33317

### CODE REVIEW

**Comment by ruudk** [comment]
Anything I can do to get this PR merged?

File: messenger.rst [hunk 1]
```diff
@@ -1223,6 +1223,8 @@ Then, make sure to "route" your message to *both* transports:
 
 That's it! You can now consume each transport:
 
+It's also possible to specify `from_transport` as attribute on the `messenger.message_handler` tag.
```
Comment (OskarStark) [inline messenger.rst]: The previous sentence ends with a `:`, so putting your sentence between the previous one and the code example looks wrong to me.

I would propose to add:

```rst
.. note::

    It is possible to specify ``from_transport`` as attribute on the ``messenger.message_handler`` tag.

    .. versionadded:: 4.4
        
        The ability to specify ``from_transport`` on the ``messenger.message_handler`` tag
        itself, was added in Symfony 4.4.
```

@javiereguiluz / @xabbuh would you agree to add the `versionadded` to the note itself?

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: messenger.rst (hunk 1 above)
Comment (ruudk) [reply messenger.rst]: Applied the change.

**Review by dbrumann — APPROVED** [review]

**Comment by wouterj** [comment]
Hi @ruudk! Thanks for submitting a doc PR alongside your feature PR and congratz on getting it merged!

I've now merged this pull requests as well. I moved the docs you added to the section explaining the tag. I think it's a bit more usefull to have it there. I've now also explicitly added a list of options that can be configured, as I couldn't find it anywhere in the docs (and it seems like the comment "options returned by getHandledMessages() are supported here" wasn't true for at least Symfony <4.4).

You can see the changes in b8526c3. Please let me know if you disagree :)

**Pull request merged**